modules/machineset-gcp-pd-disk-types.adoc

```diff
@@ -1,30 +1,50 @@
 // Module included in the following assemblies:
 //
 // * machine_management/creating_machinesets/creating-machineset-gcp.adoc
+// * machine_management/control_plane_machine_management/cpmso-using.adoc
+
+ifeval::["{context}" == "cpmso-using"]
+:cpmso:
+endif::[]
 
 :_content-type: PROCEDURE
 [id="machineset-gcp-pd-disk-types_{context}"]
-= Configuring persistent disk types by using compute machine sets
+= Configuring persistent disk types by using machine sets
 
-You can configure the type of persistent disk that a compute machine set deploys machines on by editing the compute machine set YAML file. 
+You can configure the type of persistent disk that a machine set deploys machines on by editing the machine set YAML file.
 
 For more information about persistent disk types, compatibility, regional availability, and limitations, see the GCP Compute Engine documentation about link:https://cloud.google.com/compute/docs/disks#pdspecs[persistent disks].
 
 .Procedure
 
-. In a text editor, open the YAML file for an existing compute machine set or create a new one.
+. In a text editor, open the YAML file for an existing machine set or create a new one.
 
 . Edit the following line under the `providerSpec` field:
 +
 [source,yaml]
 ----
-providerSpec:
-  value:
-    disks:
-      type: <pd-disk-type> <1>
+apiVersion: machine.openshift.io/v1
```

Comment:
MachineSets are `machine.openshift.io/v1beta1` so need to move this inside the if statement

Reply:
aaaaah very good, thanks!